libqtile/hook.py

```diff
@@ -1024,6 +1024,14 @@ def hooked_function():
         """,
         _user_hook_func,
     ),
+    Hook(
+        "plasma_add_mode",
```

Comment:
can we do this from inside plasma.py, something like:

```
from libqtile.hook import hooks
...
hooks.register_hook(Hook(...))
```
?

Reply:
that is, layouts/plasma.py

Reply:
Yes - that would be neat. I don't think it will appear in the docs but that's fine as there's no real reason for users to need the hook.